snap/hooks/install

```diff
@@ -18,3 +18,13 @@ supported.
 EOF
 	exit 1
 fi
+
+
+# Create directories required by FRR
```

Comment:
You are right, thanks, I did not realize that.

Reply:
As part of this move, I'll change the `cp -r` below to `cp -r --update=none`, to avoid overwriting existing config files on snap refresh

Reply:
I ended up doing this step both in `install` and in `post-refresh` hook, because it turned out that the `post-refresh` is not executed on fresh install.